[2025-09-16T07:14:32.724+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: opensky_data_streaming_dag.wait_8_hours manual__2025-09-16T07:14:31.554404+00:00 [queued]>
[2025-09-16T07:14:32.726+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: opensky_data_streaming_dag.wait_8_hours manual__2025-09-16T07:14:31.554404+00:00 [queued]>
[2025-09-16T07:14:32.727+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-09-16T07:14:32.730+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): wait_8_hours> on 2025-09-16 07:14:31.554404+00:00
[2025-09-16T07:14:32.732+0000] {standard_task_runner.py:60} INFO - Started process 1544 to run task
[2025-09-16T07:14:32.733+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'opensky_data_streaming_dag', 'wait_8_hours', 'manual__2025-09-16T07:14:31.554404+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/opensky_data_streaming_dag.py', '--cfg-path', '/tmp/tmp6hvm2i7f']
[2025-09-16T07:14:32.734+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask wait_8_hours
[2025-09-16T07:14:32.752+0000] {task_command.py:423} INFO - Running <TaskInstance: opensky_data_streaming_dag.wait_8_hours manual__2025-09-16T07:14:31.554404+00:00 [running]> on host 7b6d608462af
[2025-09-16T07:14:32.776+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='opensky_data_streaming_dag' AIRFLOW_CTX_TASK_ID='wait_8_hours' AIRFLOW_CTX_EXECUTION_DATE='2025-09-16T07:14:31.554404+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-16T07:14:31.554404+00:00'
[2025-09-16T07:14:32.777+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-16T07:14:32.777+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'sleep 28800']
[2025-09-16T07:14:32.780+0000] {subprocess.py:86} INFO - Output:
[2025-09-16T07:20:30.717+0000] {local_task_job_runner.py:302} WARNING - State of this instance has been externally set to success. Terminating instance.
[2025-09-16T07:20:30.719+0000] {process_utils.py:131} INFO - Sending 15 to group 1544. PIDs of all processes in the group: [1545, 1544]
[2025-09-16T07:20:30.719+0000] {process_utils.py:86} INFO - Sending the signal 15 to group 1544
[2025-09-16T07:20:30.720+0000] {taskinstance.py:2450} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-09-16T07:20:30.720+0000] {subprocess.py:104} INFO - Sending SIGTERM signal to process group
[2025-09-16T07:20:30.738+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=1544, status='terminated', exitcode=0, started='07:14:32') (1544) terminated with exit code 0
[2025-09-16T07:20:30.739+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=1545, status='terminated', started='07:14:32') (1545) terminated with exit code None
